src/Orleans/Runtime/GrainReference.cs

```diff
@@ -378,7 +398,7 @@ private void CallClientInvokeCallback(InvokeMethodRequest request)
         }
 
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes")]
-        private static void ResponseCallback(Message message, TaskCompletionSource<object> context)
+        private void ResponseCallback(Message message, TaskCompletionSource<object> context)
```

Comment:
what's non-static about this method? Am I not seeing a field access here? Maybe mark them with `this.`